src/lib/icons.js

```diff
@@ -1,35 +1,54 @@
-// Fallback to get the item icon and avoid error if not found
-// with a possible default icon
+const mimeTypes = {
```

Comment:
We are repeating ourselves since we already this library elsewhere in cozy (at least cozy-bar) :/
If we need this library again, we should export it to centralise it.

Reply:
Maybe this fetchIcon should be a wrapper in the `<AppIcon/>` like doing `ensureMimeType(fetchIcon())` and using this way, this library will be handled by the `<AppIcon/>` component and why not the generic `<Icon />` component